File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis.VisualBasic/UnaryOperatorKind.cs
using System;

namespace Microsoft.CodeAnalysis.VisualBasic
{
	[Flags]
	internal enum UnaryOperatorKind
	{
		Plus = 1,
		Minus = 2,
		Not = 3,
		IntrinsicOpMask = 3,
		Lifted = 4,
		UserDefined = 8,
		Implicit = 0x10,
		Explicit = 0x20,
		IsTrue = 0x30,
		IsFalse = 0x40,
		OpMask = 0x73,
		Error = 0x80
	}
}

File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis/VisualBasic/UnaryOperatorKind.vb
Imports System

Namespace Microsoft.CodeAnalysis.VisualBasic
	<Flags>
	Friend Enum UnaryOperatorKind
		Plus = 1
		Minus = 2
		IntrinsicOpMask = 3
		[Not] = 3
		Lifted = 4
		UserDefined = 8
		Implicit = 16
		Explicit = 32
		IsTrue = 48
		IsFalse = 64
		OpMask = 115
		[Error] = 128
	End Enum
End Namespace